experimental/mocks/mock_account_client.go

```diff
@@ -0,0 +1,253 @@
+// Code generated from OpenAPI specs by Databricks SDK Generator. DO NOT EDIT.
+
+package mocks
+
+import (
+	"github.com/databricks/databricks-sdk-go"
+	"github.com/stretchr/testify/mock"
+
+	"github.com/databricks/databricks-sdk-go/config"
+	"github.com/databricks/databricks-sdk-go/experimental/mocks/service/billing"
+	"github.com/databricks/databricks-sdk-go/experimental/mocks/service/catalog"
+	"github.com/databricks/databricks-sdk-go/experimental/mocks/service/iam"
+	"github.com/databricks/databricks-sdk-go/experimental/mocks/service/oauth2"
+	"github.com/databricks/databricks-sdk-go/experimental/mocks/service/provisioning"
+	"github.com/databricks/databricks-sdk-go/experimental/mocks/service/settings"
+)
+
+// NewAccountClient creates new mocked version of Databricks SDK client for Accounts
+// which can be used for testing.
+func NewAccountClient(t interface {
+	mock.TestingT
+	Cleanup(func())
+}, cfg *config.Config) *databricks.AccountClient {
```

Comment:
Given that the function signature is different from `databricks.NewAccountClient`, can we get rid of the `cfg` argument entirely? If users need to set it for some reason they can poke at the struct directly, and it keeps tests a little bit tighter to not have to worry about the `cfg` instance.